agent/src/podman/podman_runtime_config.rs

```diff
@@ -53,3 +55,56 @@ impl From<TryFromWorkloadSpecError> for WorkloadError {
         WorkloadError::StartError(value.0)
     }
 }
+
+#[derive(Debug, serde::Deserialize)]
+pub struct Mount {
+    #[serde(default)]
+    pub destination: Option<String>,
```

Comment:
Isn't the destination required in all cases?

Reply:
At the moment, yes destination is always required. Also, I can't image some mount without destination. But, the podman documentation, does not state that destination is always required. It only says that each mount type can have different options and then list these options. It it also an `Option` in podman_api.

Reply:
I have now changed `destination` to `String`.